provider.go

```diff
@@ -37,6 +38,15 @@ func provider() terraform.ResourceProvider {
 					return "credential-store", nil
 				},
 			},
+			"profile": {
+				Type:        schema.TypeString,
+				Required:    true,
+				DefaultFunc: func() (interface{}, error) {
```

Comment:
If you set AWS_PROFILE environment variable it drives Terraform itself nuts + I think it's just nicer to set the provider as: 
```
provider "credstash" {
  table  = "some-table"
  region = "some-region"
  profile = "some-profile" 
}
```